index.js

```diff
@@ -29,6 +29,14 @@ export let Cell = {
       classes.push('pure-u-x-large-' + props.xlgSize);
     }
 
-    return (<div class={classes}>{props.children}</div>);
+    return (<div class={[classes, propsClasses(props)]}>{props.children}</div>);
   }
 };
+
+function propsClasses (props) {
+  if (!props.class) {
+    return '';
+  }
+
+  return props.class.split(' ');
```

Comment:
we're returning an array if props.class is set but a String otherwise, why? Also, I'dd do `return (props.class || '').split(' ')` just to keep the chars down :)